### PR TITLE
add description for task scheduler

### PR DESCRIPTION
**Pull request by fzhedu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add description for task scheduler, which has been recently added in tiflash_summary.json.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/monitor-tiflash.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
 - Cop task memory usage：所有 TiFlash 实例处理 coprocessor 请求占用的总内存。
 - Handling Request Number：所有 TiFlash 实例正在处理的 coprocessor 请求数量之和。请求的分类与 Request QPS 中的相同.
 
+## Task Scheduler
+
+- Min TSO：每个 TiFlash 实例上正在运行的查询语句中 TSO 最小值，默认值是 Uint64 的最大值。正常负载运行情况下，min TSO 不断在演进，呈现出一条斜向上的直线。如果某一阶段变成水平线段，表示这一段时间内都一直是当前 TSO 的查询在运行。更坏情况是当前有效 min TSO 一直不变，表示当前 TSO 对应的查询 hang 住，需要结合日志进一步排查。
+- Estimated Thread Usage and Limit：每个 TiFlash 实例上当前正在运行的所有任务占用的估计的线程使用量，以及该实例上任务调度器设置的估计线程使用量的软限制和硬限制。
+- Active and Waiting Queries Count：每个 TiFlash 实例上当前正在运行的查询和等待的查询数目。
+- Active and Waiting Tasks Count：每个 TiFlash 实例上当前正在运行的任务和等待的任务数目。
+- Hard Limit Exceeded Count：每个 TiFlash 实例上正在运行的任务的估计的线程使用量超过硬限制的次数，正常情况下是 0，非 0 情况有查询报错，需要进行排查。
```
Comment (zanmato1984) [inline tiflash/monitor-tiflash.md:53]: 为何非 0 需要排查？有可能就是并发过多导致超限了吧？

File: tiflash/monitor-tiflash.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
 - Cop task memory usage：所有 TiFlash 实例处理 coprocessor 请求占用的总内存。
 - Handling Request Number：所有 TiFlash 实例正在处理的 coprocessor 请求数量之和。请求的分类与 Request QPS 中的相同.
 
+## Task Scheduler
+
+- Min TSO：每个 TiFlash 实例上正在运行的查询语句中 TSO 最小值，默认值是 Uint64 的最大值。正常负载运行情况下，min TSO 不断在演进，呈现出一条斜向上的直线。如果某一阶段变成水平线段，表示这一段时间内都一直是当前 TSO 的查询在运行。更坏情况是当前有效 min TSO 一直不变，表示当前 TSO 对应的查询 hang 住，需要结合日志进一步排查。
```
Comment (zanmato1984) [inline tiflash/monitor-tiflash.md:49]:
```suggestion
- Min TSO：每个 TiFlash 实例上正在运行的查询语句中 TSO 最小值，TiFlash 会维护该值以确保具有更小 TSO 的查询可以被调度到。如果当前没有正在运行的查询，则该值为 `uint64` 最大值。
```

Comment (zanmato1984) [reply tiflash/monitor-tiflash.md:49]: 这里仅做概念性描述即可。

File: tiflash/monitor-tiflash.md (hunk 1 above)
Comment (fzhedu) [reply tiflash/monitor-tiflash.md:53]: 有可能是 TSO 逆序过多，也可能 minTSO 查询占用线程数太多，hard limit 都容纳不下

Comment (zanmato1984) [reply tiflash/monitor-tiflash.md:53]: 跟前面第一条一样的原则，这里只写概念，涉及到可能的问题和排查不放到这里了吧。

**Review by zanmato1984 — APPROVED** [review]
LGTM

File: tiflash/monitor-tiflash.md (hunk 1 above)
Comment (fzhedu) [reply tiflash/monitor-tiflash.md]: updated.

File: tiflash/monitor-tiflash.md [hunk 3]
```diff
@@ -44,6 +44,15 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
 - Handling Request Number：所有 TiFlash 实例正在处理的 coprocessor 请求数量之和。请求的分类与 Request QPS 中的相同.
 
+## Task Scheduler
+
+- Min TSO：每个 TiFlash 实例上正在运行的查询语句中 TSO 最小值，TiFlash 会维护该值以确保具有更小 TSO 的查询可以被调度到。如果当前没有正在运行的查询，则该值为 `uint64` 最大值。
+- Estimated Thread Usage and Limit：每个 TiFlash 实例上当前正在运行的所有任务占用的估计的线程使用量，以及该实例上任务调度器设置的估计线程使用量的软限制和硬限制。
+- Active and Waiting Queries Count：每个 TiFlash 实例上当前正在运行的查询和等待的查询数目。
+- Active and Waiting Tasks Count：每个 TiFlash 实例上当前正在运行的任务和等待的任务数目。
+- Hard Limit Exceeded Count：每个 TiFlash 实例上正在运行的任务的估计的线程使用量超过硬限制的次数。
+- Task Waiting Duration：每个 TiFlash 实例上任务初始化之后等待被调度的时长。
```
Comment (shichun-0415) [inline tiflash/monitor-tiflash.md:54]:
```suggestion
- Min TSO：每个 TiFlash 实例上正在运行的查询语句中的最小 TSO，该参数确保具有最小 TSO 的查询可以被调度。如果当前没有正在运行的查询，则该值为最大值 `uint64`。
- Estimated Thread Usage and Limit：每个 TiFlash 实例上正在运行的所有任务占用的线程估值，以及该实例上任务调度器设置的估算线程用量的软限制和硬限制。
- Active and Waiting Queries Count：每个 TiFlash 实例上正在运行的查询数量和等待的查询数量。
- Active and Waiting Tasks Count：每个 TiFlash 实例上正在运行的任务数量和等待的任务数量。
- Hard Limit Exceeded Count：每个 TiFlash 实例上运行中任务的估算线程用量超过硬限制的次数。
- Task Waiting Duration：每个 TiFlash 实例上任务从初始化到被调度的等待时长。
```

**Review by shichun-0415 — APPROVED** [review]

File: tiflash/monitor-tiflash.md [hunk 4]
```diff
@@ -44,6 +44,15 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
 - Cop task memory usage：所有 TiFlash 实例处理 coprocessor 请求占用的总内存。
 - Handling Request Number：所有 TiFlash 实例正在处理的 coprocessor 请求数量之和。请求的分类与 Request QPS 中的相同.
 
+## Task Scheduler
+
+- Min TSO：每个 TiFlash 实例上正在运行的查询语句中的最小 TSO，该值确保具有最小 TSO 的查询可以被调度。如果当前没有正在运行的查询，则该值为 `uint64` 整数型最大值。
+- Estimated Thread Usage and Limit：每个 TiFlash 实例上正在运行的所有任务占用的线程估值，以及该实例上任务调度器设置的估算线程用量的软限制和硬限制。
+- Active and Waiting Queries Count：每个 TiFlash 实例上正在运行的查询数量和等待的查询数量。
+- Active and Waiting Tasks Count：每个 TiFlash 实例上正在运行的任务数量和等待的任务数量。
```
Comment (qiancai) [inline tiflash/monitor-tiflash.md:52]:
```suggestion
- Active and Waiting Queries Count：每个 TiFlash 实例上正在运行的查询数量和正在等待的查询数量。
- Active and Waiting Tasks Count：每个 TiFlash 实例上正在运行的任务数量和正在等待的任务数量。
```

File: tiflash/monitor-tiflash.md [hunk 5]
```diff
@@ -44,6 +44,15 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
+- Hard Limit Exceeded Count：每个 TiFlash 实例上运行中任务的估算线程用量超过硬限制的次数。
```
Comment (qiancai) [inline tiflash/monitor-tiflash.md:53]: 这个 count 等于 Active Tasks Count 减去  Thread Hardlimt 吗

**Review by qiancai — COMMENTED** [review]
@fzhedu PTAL 谢谢

File: tiflash/monitor-tiflash.md (hunk 5 above)
Comment (zanmato1984) [reply tiflash/monitor-tiflash.md:53]: 这几个数字是没有关系的。如果一条query被调度器调度执行，就会算到active，如果被调度器命令等待，就会算到waiting，如果被调度器认为超过hard limit拒绝执行，就会把这个count加1，然后这条query会失败。

Comment (qiancai) [reply tiflash/monitor-tiflash.md:53]: 哦哦好的。对了，请问硬限制是指什么的硬限制呢

Comment (zanmato1984) [reply tiflash/monitor-tiflash.md:53]: `Estimated Thread Usage and Limit：每个 TiFlash 实例上正在运行的所有任务占用的线程估值，以及该实例上任务调度器设置的估算线程用量的软限制和硬限制。`

Comment (qiancai) [reply tiflash/monitor-tiflash.md:53]:
```suggestion
- Hard Limit Exceeded Count：每个 TiFlash 实例上运行中任务的估算线程用量超过了设置的硬限制的次数。
```

Comment (qiancai) [reply tiflash/monitor-tiflash.md:53]: PTAL

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**